src/Book.java

```diff
@@ -0,0 +1,29 @@
+public class Book {
+    private String bookName;
+    private String author;
```

Comment:
автор это автор, а не строка. Нужно использовать правильный тип
```suggestion
    private Author author;
```